part2/stages/Functions/install-main

```diff
@@ -895,11 +894,6 @@ is_mounted()
 {
     local MOUNT_PT=$1
     cat /proc/mounts | awk '{ print $2 }' | grep -q "^${MOUNT_PT}$"
-    if [ $? -eq 0 ]; then
-        return 1
-    else
-        return 0
-    fi
```

Comment:
Yeah thanks for removing that atrocity... Shouldn't we also output to /dev/null?